js/members.js

```diff
@@ -83,3 +135,12 @@ function clickThirdYears() {
 function clickSecondYears() {
     $('#allMembersContainer').html(members['secondYears']);
 }
+
+function clickAllAlumni() {
```

Comment:
Create a new function ```click2018Alumni()``` and move the entire contents of this function there. Inside this function, simply call ```click2018Alumni()```.

This will improve future support for more alumni batches.